src/WorkItemMigrator/JiraExport/JiraProvider.cs

```diff
@@ -116,13 +116,15 @@ private async Task<JiraAttachment> DownloadAttachmentAsync(JiraAttachment att, W
                 {
                     try
                     {
-                        string path = Path.Combine(Settings.AttachmentsDir, att.Id, att.Filename);
+                        var path = Path.Combine(Settings.AttachmentsDir, att.Id, att.Filename);
                         EnsurePath(path);
-                        await web.DownloadWithAuthenticationAsync(att.Url, path);
+
+                        await DownloadWithJiraRestClientAsync(att.Url, path);
+
                         att.LocalPath = path;
                         Logger.Log(LogLevel.Debug, $"Downloaded attachment '{att.ToString()}'");
                     }
-                    catch (Exception)
+                    catch (Exception e)
```

Comment:
Small Code smell as pointed out by sonarqube, let's log the e or dismiss it, 
personally i would log it.

Reply:
Thanks for the input :)